npc/re/instances/horror_toy_factory.txt

```diff
@@ -0,0 +1,1984 @@
+	end;
+
+OnInstanceInit:
+	disablenpc(instance_npcname("#finalbs_e"));
+	end;
+
+OnStart:
+	.@in_map$ = instance_mapname("1@xm_d");
+	mapannounce(.@in_map$, _("Celine Kimi's Scream: It's futile to defeat me! I'll just come back to life again and again!"), bc_map, 0xff6666, FW_NORMAL, 15);
+	enablenpc(instance_npcname("#fac6wp"));
+	enablenpc(instance_npcname("#jeton2"));
+	enablenpc(instance_npcname("Catherine Zeta Jones#6"));
+	for (.@i = 1; .@i <= 10; ++.@i)
+		enablenpc(instance_npcname("Gift Box#"+.@i));
+	sleep(6000);
+	mapannounce(.@in_map$, _("Catherine Zeta Jones: P..please run towards the south exit!"), bc_map, C_YELLOW, FW_NORMAL, 12);
+	end;
+}
+
+1@xm_d,205,159,0	script	#fac6wp	WARPNPC,2,2,{
+	end;
+
+OnInstanceInit:
+	disablenpc(instance_npcname("#fac6wp"));
+	end;
+
+OnTouch_:
+	warp(instance_mapname("1@xm_d"), 205, 147);
+	end;
+}
+
+1@xm_d,218,145,0	script	#jeton2	HIDDEN_WARP_NPC,4,4,{
+	end;
+
+OnInstanceInit:
+	disablenpc(instance_npcname("#jeton1"));
+	end;
+
+OnTouch:
+	disablenpc(instance_npcname("#jeton2"));
+	donpcevent(instance_npcname("Catherine Zeta Jones#6")+"::OnStart");
+	end;
+}
+
+1@xm_d,218,150,5	script	#exwp1	PORTAL,{
+	mes("Do you want to leave?");
+	next();
+	if (select("Just a little while more", "Leave") == 1) {
+		mes("The machine stops operating.");
+		close();
+	}
+	close2();
+	warp("xmas", 233, 300);
+	end;
+
+OnInstanceInit:
+	disablenpc(instance_npcname("#exwp1"));
+	end;
+}
+
+1@xm_d,218,145,5	script	Catherine Zeta Jones#6	4_F_SKULL06GIRL,{
+	end;
+
+OnInstanceInit:
+	disablenpc(instance_npcname("Catherine Zeta Jones#6"));
+	end;
+
+OnStart:
+	sleep(1000);
+	npctalk(_("Catherine Zeta Jones: It's a shame we d..didn't get to convince Kimi.."));
+	sleep(3000);
+	npctalk(_("Catherine Zeta Jones: W..what was the motive of that illusion..? W..why does it constantly try to hurt Kimi?"));
+	sleep(4000);
+	npctalk(_("Catherine Zeta Jones: Could it b..be that my appearance was a curse casted by them?"));
+	sleep(4000);
+	npctalk(_("Catherine Zeta Jones: B..but even though Kimi resents everyone, she still has unlingering feelings for this p..place."));
+	sleep(6000);
+	npctalk(_("Catherine Zeta Jones: W..we should keep everything t..tidy so as to not disappoint Kimi's s..spirit once she returns.."));
+	sleep(5000);
+	npctalk(_("Catherine Zeta Jones: It's a relief t..that the truth is finally o..out.. If there's another chance to h..help Kimi move on to her next l..life.. Do let me k..know.."));
+	sleep(5000);
+	npctalk(_("Catherine Zeta Jones: I will open the exit n..now.. Please follow me!"));
+	sleep(3000);
+	disablenpc(instance_npcname("Catherine Zeta Jones#6"));
+	enablenpc(instance_npcname("#exwp1"));
+	end;
+}
+
+1@xm_d,210,141,3	script	Gift Box#1	4_TREASURE_BOX,{
+	.@in_npc$ = instance_npcname(strnpcinfo(NPC_NAME));
+	specialeffect(EF_COIN, AREA, .@in_npc$);
+	disablenpc(.@in_npc$);
+	initnpctimer();
+	end;
+
+OnTimer1000:
+	switch (atoi(strnpcinfo(NPC_NAME_HIDDEN))) {
+	case 1:
+		.@idnum001 = rand(4, 8);
```

Comment:
what is this `.@idnu001` for? seem like not used anywhere. Or you meant `.@item_count` instead ?